csharp/ql/test/library-tests/frameworks/EntityFramework/FlowSummaries.expected

```diff
@@ -1,134 +1,18 @@
-| System.Data.Entity.DbContext.SaveChangesAsync() | property Street of element of property Addresses of argument -1 -> property Street of element of property Addresses of element of return (jump to get_Persons) | true |
-| System.Data.Entity.DbContext.SaveChangesAsync() | property Street of element of property Addresses of argument -1 -> property Street of element of return (jump to get_Addresses) | true |
-| System.Data.Entity.DbContext.SaveChangesAsync() | property Street of element of property Addresses of argument -1 -> property Street of property Address of element of return (jump to get_PersonAddresses) | true |
-| System.Data.Entity.DbContext.SaveChangesAsync() | property Street of element of property Addresses of element of property Persons of argument -1 -> property Street of element of property Addresses of element of return (jump to get_Persons) | true |
-| System.Data.Entity.DbContext.SaveChangesAsync() | property Street of element of property Addresses of element of property Persons of argument -1 -> property Street of element of return (jump to get_Addresses) | true |
-| System.Data.Entity.DbContext.SaveChangesAsync() | property Street of element of property Addresses of element of property Persons of argument -1 -> property Street of property Address of element of return (jump to get_PersonAddresses) | true |
-| System.Data.Entity.DbContext.SaveChangesAsync() | property Street of property Address of element of property PersonAddresses of argument -1 -> property Street of element of property Addresses of element of return (jump to get_Persons) | true |
-| System.Data.Entity.DbContext.SaveChangesAsync() | property Street of property Address of element of property PersonAddresses of argument -1 -> property Street of element of return (jump to get_Addresses) | true |
-| System.Data.Entity.DbContext.SaveChangesAsync() | property Street of property Address of element of property PersonAddresses of argument -1 -> property Street of property Address of element of return (jump to get_PersonAddresses) | true |
-| System.Data.Entity.DbSet<>.Add(T) | argument 0 -> element of argument -1 | true |
-| System.Data.Entity.DbSet<>.AddAsync(T) | argument 0 -> element of argument -1 | true |
-| System.Data.Entity.DbSet<>.AddRange(IEnumerable<T>) | element of argument 0 -> element of argument -1 | true |
-| System.Data.Entity.DbSet<>.AddRangeAsync(IEnumerable<T>) | element of argument 0 -> element of argument -1 | true |
-| System.Data.Entity.DbSet<>.Attach(T) | argument 0 -> element of argument -1 | true |
-| System.Data.Entity.DbSet<>.AttachRange(IEnumerable<T>) | element of argument 0 -> element of argument -1 | true |
-| System.Data.Entity.DbSet<>.Update(T) | argument 0 -> element of argument -1 | true |
-| System.Data.Entity.DbSet<>.UpdateRange(IEnumerable<T>) | element of argument 0 -> element of argument -1 | true |
+| Microsoft.EntityFrameworkCore;DbSet<>;false;Add;(T);;Argument[0];Element of Argument[-1];value |
```

Comment:
@hvitved : We loose all the "jump to" information from the returnkind. Is this acceptable or can it somehow be encoded using the flow summary syntax? Or should we leave this test alone and print it in the "old" format?

Reply:
Printing will be similar to the flow summary syntax, but it will not parse as an actual flow summary,